core/runtime/src/main/java/io/quarkus/runtime/logging/LoggingSetupRecorder.java

```diff
@@ -222,9 +223,10 @@ public void accept(String categoryName, CategoryConfig config) {
                             CategoryBuildTimeConfig::getMinLevel, categoryDefaultMinLevels, buildConfig.minLevel);
 
                     if (logLevel.intValue() < minLogLevel.intValue()) {
-                        log.warnf("Log level %s for category '%s' set below minimum logging level %s, promoting it to %s",
+                        log.warnf(
+                                "Log level %s for category '%s' set below minimum logging level %s, promoting it to %s. Otherwise you will need to set the build time configuration property 'quarkus.log.category.\"%s\".level' to '%s'",
```

Comment:
```suggestion
                                "Log level %s for category '%s' set below minimum logging level %s, promoting it to %s. Set the build time configuration property 'quarkus.log.category.\"%s\".level' to '%s' to avoid this warning",
```

Reply:
Fixed here and in the other instances